ratis-server/src/main/java/org/apache/ratis/server/impl/RaftConfigurationImpl.java

```diff
@@ -63,10 +63,6 @@ Builder setConf(PeerConfiguration conf) {
       return this;
     }
 
-    Builder setConf(Iterable<RaftPeer> peers) {
-      return setConf(new PeerConfiguration(peers));
-    }
-
```

Comment:
Let's keep this builder method since ServerState has a valid use of it.  Then, we don't have to change ServerState, DataStreamBaseTest and some other code.